docs/dev/index.md

```diff
@@ -22,16 +21,17 @@ This document provides an index to all development guidelines and background inf
 5. Insert the following commands  `docker-compose up -d`
 
 #### IntelliJ
-3. [Follow PatternPediaAPI from Step 5 pls](#step5) 
+3. [Follow PatternPediaAPI from Step 5 pls](#step5)
 
+#### Turn On/Off
```

Comment:
Turn On/Off for what? --> I think the new explanations belong to PatternPedia Auth? 
Please also add a few sentences that explain what this component is about

Reply:
I added a short description and changed the on/off so it hopefully makes more sense